package.json

```diff
@@ -85,12 +85,13 @@
     "@storybook/addons": "7.6.17",
     "@storybook/react": "7.6.17",
     "@storybook/react-webpack5": "7.6.17",
+    "@testing-library/jest-dom": "5.17.0",
```

Comment:
any reason we can't use v6?

Reply:
I tried using v6 but it tests were not passing with it's peer dependency version of jest

Reply:
Part of the problem is we currently extend `jest-environment-jsdom` which works with jest v27 but upgrading to a higher version of jest would require us to use `jsdom` which I could not extend in the same manner.

Reply:
Was it not as simple as importing https://www.npmjs.com/package/jest-environment-jsdom since it doesn't come bundled with jest anymore? I approved this PR already since it at least moves the version forward, but just wondering if it isn't as simple as that?

Reply:
I'll follow up to try again with the latest jest (currently getting a bunch of npm resolution errors while trying to bump back up to v29)

Reply:
I was getting this error when updateing to jest and ts-jest v29:

`TypeError: Class extends value #<Object> is not a constructor or null

       6 | const assert = require('assert')
       7 | const JSDOMEnvironment = require('jest-environment-jsdom')
    >  8 | const webcrypto = require('crypto').webcrypto
         |                                                  ^
       9 |
      10 | module.exports = class CustomEnvironment extends JSDOMEnvironment {
      11 |   async setup() {`

Reply:
I got past the class extension using

`
module.exports = class CustomEnvironment extends (
  JSDOMEnvironment.TestEnvironment
) {
`

but now window.crypto.randomUUID is apparently no longer a function